### PR TITLE
Multiple media sources

### PR DESCRIPTION
**Pull request by Zimbardoex**
#### Solves:
- The Add support for multiple media sources milestone
- #45 
- #248
- #250 
- #251 
- #252 
- #253
- #258
- #259 
- #265 
- #271
- #278
- #281 
- #309 
- #321 

[Code review checklist](https://github.com/op-analytics/NYT-Media-Analytics/wiki/Code-Review-Checklist)

### CODE REVIEW

**Comment by NICHTJ3** [comment]
Lets never do a branch this big again... My lordy 🥇 🐙 🌊

File: front-end/src/components/LineCharts/hooks/useStyles.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core/styles';
```
[the remark this answers is not in the document]
Comment (Zimbardoex) [reply front-end/src/components/LineCharts/hooks/useStyles.js:1]: Good call, done

File: front-end/src/modules/shared/errorHelpers.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ function createGeneralError(errorMessage) {
 }
 
 export function getErrorsFromResponse(response) {
```
[the remark this answers is not in the document]
Comment (Zimbardoex) [reply front-end/src/modules/shared/errorHelpers.js:5]: Errors gets set correctly if that's what you are asking?

File: front-end/src/components/CsvDownloadButton.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import Fab from '@material-ui/core/Fab';
```
[the remark this answers is not in the document]
Comment (Zimbardoex) [reply front-end/src/components/CsvDownloadButton.js:1]: Do you mean other branches/issues? This is on dev already.

File: express-back-end/src/app.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ import db from './db';
 
 const app = express();
 
-// Stay safe where a helmet
+// Stay safe, wear a helmet
```
[the remark this answers is not in the document]
Comment (Zimbardoex) [reply express-back-end/src/app.ts:14]: Learn to spel :disappointed:

File: front-end/src/components/CsvDownloadButton.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (NICHTJ3) [reply front-end/src/components/CsvDownloadButton.js:1]: Yeah the other issues... There might be a lot

[the remark this answers is not in the document]
Comment (Zimbardoex) [reply front-end/src/components/CsvDownloadButton.js:1]: Added all open issues and anything explicitly referenced

File: front-end/src/modules/shared/errorHelpers.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zimbardoex) [reply front-end/src/modules/shared/errorHelpers.js:5]: Howzat!! :scream_cat:

**Pull request merged**